tests/installation/partitioning_expert.pm

```diff
@@ -0,0 +1,36 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+#
+# Summary: 
+# Maintainer: Christopher Hofmann <cwh@suse.de> 
+
+use strict;
+use warnings;
+use base "y2logsstep";
+use testapi;
+
+sub run() {
+    send_key $cmd{expertpartitioner};
+    assert_screen 'expert-partitioner';
+
+    assert_and_click 'hard-disks';
+    assert_and_click 'home';
+
+    send_key 'alt-d'; # Delete
```

Comment:
I guess this might fail because the next keypress might get lost. Better guard it with `wait_screen_change` -> `wait_screen_change { send_key 'alt-d' };`